003-quarkus-many-extensions/pom.xml

```diff
@@ -201,11 +195,15 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-spring-security</artifactId>
         </dependency>
-        <!-- java.lang.RuntimeException: java.lang.RuntimeException: java.lang.IllegalArgumentException: Multiple matching properties for name "datasource.url" property was matched by both public java.util.Optional io.quarkus.reactive.pg.client.runtime.DataSourceConfig.url and public java.util.Optional io.quarkus.agroal.runtime.DataSourceRuntimeConfig.url. This is likely because you have an incompatible combination of extensions that both define the same properties (e.g. including both reactive and blocking database extensions) -->
-        <!-- <dependency>
-          <groupId>io.quarkus</groupId>
-          <artifactId>quarkus-reactive-pg-client</artifactId>
-        </dependency> -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-reactive-pg-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
```

Comment:
IMHO this is a wrong approach. Testcontainer and `@WithTestResource` etc. are heavily tested upstream. Users are supposed to use Dev Services or OIDC Wiremock test resource, let's use them unless there is a good reason. It's harder to maintain and will take are more time then setting couple of dev svc config properties.